jekyll/_cci2/configuration-reference.md

```diff
@@ -31,7 +31,7 @@ Key | Required | Type | Description
 docker | Y <sup>(1)</sup> | List | Options for [docker executor](#docker-executor)
 machine | Y <sup>(1)</sup> | Map | Options for [machine executor](#machine-executor)
 steps | Y | List | A list of [steps](#steps) to be performed
-working_directory | Y | String | What directory to run the steps in (default: depends on executor)
+working_directory | Y | String | What directory to run the steps in. Can be short-handed as `workDir`.
```

Comment:
In that case, I think instead of "Can be short-handed as `workDir`", something like "previously `workDir`" or something similar is better. Still leaves a trail for people coming from older syntax, but doesn't appear to promote or support its use.